code/iaas/allocator/src/main/java/io/cattle/platform/allocator/service/impl/AllocatorServiceImpl.java

```diff
@@ -205,6 +206,16 @@ public void volumeDeallocate(Volume volume) {
     }
 
     @Override
+    public void newPortAllocateForLoadBalancerUpdate(Instance instance, List<Port> newPorts) {
+        callExternalSchedulerForNewPorts(instance, newPorts);
```

Comment:
Lets name the methods as:

callExternalScheduleToAllocatePorts
callExternalSchedulerToRemovePorts